Core/include/Acts/Seeding/PathSeeder.hpp

```diff
@@ -9,8 +9,18 @@
 #pragma once
 
 #include "Acts/EventData/SourceLink.hpp"
+#include "Acts/EventData/TrackParameters.hpp"
+#include "Acts/Seeding/detail/UtilityFunctions.hpp"
 #include "Acts/Surfaces/Surface.hpp"
 #include "Acts/Utilities/Delegate.hpp"
+#include "Acts/Utilities/GridIterator.hpp"
+
+namespace {
+
+template <typename grid_t>
+concept SourceLinkGrid =
+    std::same_as<typename grid_t::value_type, std::vector<Acts::SourceLink>>;
+}
 
 namespace Acts::Experimental {
```

Comment:
This isn't tied to the Gen2 geometry, is it? It's fine to keep the `Experimental` namespace, but it would also be fine to drop it here.

Reply:
Yes, it is geometry independent. Removed `Experimental`